cicd/shared.sh

```diff
@@ -228,7 +228,7 @@ check_others() {
     fi
 
     record_ext_version ai "$pg" ""
-    if [ -n "$PGAI_VERSION" ]; then
+    if [ -n "$PGAI_VERSION" ] && [ "$pg" -gt 15 ]; then
```

Comment:
I'm partial to this since we're using bash, but feel free to ignore me :)

```suggestion
    if [[ -n "$PGAI_VERSION" && "$pg" -gt 15 ]]; then
```